stackoversight/scraper/release_heap.py

```diff
@@ -0,0 +1,48 @@
+# Need a heap ofc
+import heapq
+# For release timers
+import threading
+# For the release queue
+from queue import Queue
+
+
+class ReleaseHeap(object):
```

Comment:
I notice there is ReleaseHeap(object) and ReleaseHeap(queue) (I do not know if doing (object) is standard notation or something, but it is redundant and not needed for Python 3) do both classes need to exist? If so consider changing their names to show explicit differences

Reply:
This is resolved with latest push